src/Order.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = (props) => {
+  const [orders, setOrders] = useState(0);
+  const orderOne = () => {
+    setOrders((prevOrder) => prevOrder + 1);
```

Comment:
This is currently working great however if you wish to shorten it you can just do prevOrder + 1 without requiring the arrow function